packages/OreIdWalletConnect/src/ConnectWalletWidget/ConnectWalletWidget.tsx

```diff
@@ -37,9 +37,7 @@ export const ConnectWalletWidget: React.FC<ConnectWalletWidgetProps> = ({ onClic
         <div>
           <QrCodeImage size={30} />
         </div>
-        <div>
```

Comment:
This div has a reason... Leave the spacing equal, as shown in the figma. And also ensure that CSS will not influence the internal component.

Reply:
its correct as per updated styles